lib/plugins/en_route.rb

```diff
@@ -0,0 +1 @@
+CreditCardValidations.add_brand(:en_route, {length: 15, prefixes: %w(2014 2149)}, {skip_luhn: true})
```

Comment:
Line is too long. [100/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Final newline missing.